apps/OboeTester/app/src/main/cpp/flowgraph/SineOscillator.h

```diff
@@ -14,21 +14,20 @@
  * limitations under the License.
  */
 
-#ifndef NATIVEOBOE_SINEGENERATOR_H
-#define NATIVEOBOE_SINEGENERATOR_H
+#ifndef FLOWGRAPH_SINE_OSCILLATOR_H
+#define FLOWGRAPH_SINE_OSCILLATOR_H
 
 #include <unistd.h>
 
 #include "OscillatorBase.h"
 
-class SineGenerator : public OscillatorBase {
+class SineOscillator : public OscillatorBase {
```

Comment:
Add class description here and for all classes which don't have one.

Reply:
Done